packages/web/components/Primitives/Base.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { StyledBase } from "./Styled";
+import { useTheme } from "../../../themes";
+
+export function Base(props) {
+  const theme = useTheme();
+  return <StyledBase theme={theme} m={2} {...props} />;
```

Comment:
What `m` prop means?

Reply:
Margin - https://styled-system.com/table

Reply:
🤔

Reply:
`margin` also works (m is just shorthand)